src/applications/personalization/dashboard-2/tests/e2e/dashboard-loa1.cypress.spec.js

```diff
@@ -0,0 +1,87 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
+import loa1User from '@@profile/tests/fixtures/users/user-loa1.json';
+
+import manifest from 'applications/personalization/dashboard/manifest.json';
+
+import { mockFeatureToggles } from './helpers';
+
+/**
+ *
+ * @param {boolean} mobile - test on a mobile viewport or not
+ * @param {array} stubs - array of Cy.stubs that should _not_ be called
+ *
+ * This helper:
+ * - loads the my VA Dashboard,
+ * - checks that focus is managed correctly, and performs an aXe scan
+ */
+function loa1DashboardTest(mobile, stubs) {
+  mockFeatureToggles();
+  cy.visit(manifest.rootUrl);
+
+  if (mobile) {
+    cy.viewport('iphone-4');
+  }
+
+  // focus should be on the h1
+  cy.focused()
+    .should('have.attr', 'id', 'dashboard-title')
+    .contains('My VA')
+    .and('have.prop', 'tagName')
+    .should('equal', 'H1');
+
+  // make sure that we don't call APIs to get name, service history, or disability rating
+  cy.should(() => {
+    stubs.forEach(stub => {
+      expect(stub).not.to.be.called;
```

Comment:
I confirmed that this test is working by temporarily making the Dashboard call the APIs even when the user is LOA1